main.js

```diff
@@ -10,3 +10,30 @@ document.querySelectorAll(".nav-link").forEach((n) =>
     sideMenu.classList.remove("active");
   })
 );
+
+const form = document.getElementsByTagName("form");
+const email = document.getElementById("mail");
+const emailError = document.querySelector("#mail + span.error");
+
+email.addEventListener("input", () => {
+  if (email.value.search(/[a-z]g/) !== -1) {
+    emailError.textContent = "success";
+    emailError.className = "error";
+  } else {
+    showError();
+  }
+});
+
```

Comment:
This block of code is unnecessary. You want to validate your input field after submit action. Kindly check this [tutorial](https://www.w3schools.com/js/js_validation.asp) to validate a form input